install_config/install/deploy_router.adoc

```diff
@@ -418,13 +418,14 @@ the actual remote IP address.
 
 [NOTE]
 ====
-On OpenShift cluster with
-link:../../architecture/additional_concepts/sdn.html#multitenant-plugin[multi-tenant
-network isolation], router on a non-default namespace with `--host-network=false`
-option will load all routes in the cluster but routes across the namespaces will not
-be reachable due to network isolation. With `--host-network=true` option, routes
-will bypass the container network and it can access any pod in the cluster.
-If isolation is needed in this case, then don't add routes across the namespaces.
+On OpenShift clusters using
+link:../../architecture/additional_concepts/sdn.html#network-isolation-using-the-multitenant-plugin[multi-tenant
```

Comment:
link reference should be ../../architecture/additional_concepts/sdn.html#network-isolation-multitenant (*network-isolation-multitenant* instead of *network-isolation-using-the-multitenant-plugin*)